src/GlobalStyle.js

```diff
@@ -11,12 +11,19 @@ body {
   font-size: 16px;
   margin: 0;
   padding: 0;
+  display:flex;
```

Comment:
This might work but are you sure you want everything to be a flex box unless stated otherwise ?

Reply:
This only targets the body so it should be fine. I did this to have a max width for my app. But as i am not doing this anymore it is not necessary.